views/layout.ejs

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--Copyright 2020 Google LLC
```

Comment:
Let's omit copyright notices from the generated pages. I shouldn't have included it in the first place :)